website/docs/set-up-your-environment.mdx

```diff
@@ -0,0 +1,132 @@
+import Tabs from  '@theme/Tabs'
+import TabItem from  '@theme/TabItem'
+
+# Set up your environment
+The first step to build with ComposeDB is setting up your development environment. This guide will show you how to install and configure relevant packages and tools.
+
+## Installation
+### Create your project directory
+Start with creating the project directory. Here you’ll store all your app’s local files:
+```sh
+mkdir my-project
+cd my-project
+```
+
+### Node.js
+If you don’t already have them installed, you will need to install Node.js v16 and the following package managers:
+- [NodeJS v16](https://nodejs.org/en/)
+- [pnpm v7](https://pnpm.io/installation)
+
+Not sure which version you have? Use this command to make sure you have the correct versions installed.
+```sh
+node -v
+pnpm -v
+```
+
+### Ceramic
+ComposeDB runs on Ceramic, so you will need to run a Ceramic node. To get started, and for testing purposes, we recommend running a local Ceramic node. If you're interested in running the production node, you can follow the [tutorial here](https://blog.ceramic.network/configuring-a-ceramic-node-using-aws/). 
+
+The Ceramic CLI provides a set of commands that make it easier to run and manage Ceramic nodes. We will start by installing the Ceramic CLI:
+
+```sh
+pnpm install -g @ceramicnetwork/cli
+```
+
+### ComposeDB
+
+  
+
+Next install the ComposeDB CLI, which enables you to interact with ComposeDB data from your terminal. Install ComposeDB CLI using the following command:
+
+```sh
+pnpm add -g @composedb/cli@^0.3.0
```

Comment:
Tabs with different package managers options

Reply:
noted, pinged justina - will add to next update